build.gradle

```diff
@@ -142,6 +142,14 @@ subprojects {
       from javadoc
     }
 
+    /*
+    Some methods created by lombok don't have a description set. This is going to suppress all javadoc warnings
+    and avoid our prompt to get clutter with nonessential warnings.
+    */
+    tasks.withType(Javadoc) {
+      options.addStringOption('Xdoclint:none', '-quiet')
+    }
+
```

Comment:
How is this targeting only lombok-generated files? I don't think we want to turn off linting for everything.

Reply:
Unfortunately it targets all warnings. I was not able to find a way to suppress only specific files, either notations or compilation flags, none of my attempts seemed to work. I thought it would be better to disable everything and keep it clean. I can enable it again if you think this is not the best approach.

Reply:
I removed this piece of code from the current revision.